File: reviews-service/Domain/Book.cs
using System;

namespace Domain
{
    public class Book
    {
        public int Id { get; set; }
        public string Isbn { get; set; }
        public string Title { get; set; }
        public string Authors { get; set; }
        public int Year { get; set; }
        public int Quantity { get; set; }
        public int OrganizationId { get; set; }
        public int TimesReserved { get; set; }
        public bool Deleted { get; set; }
    }
}

File: books-service/models/book.js
'use strict';
const {
  Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
  class Book extends Model {
    /**
     * Helper method for defining associations.
     * This method is not a part of Sequelize lifecycle.
     * The `models/index` file will call this method automatically.
     */
    static associate(models) {
      // define association here
    }
  };
  Book.init({
    isbn: DataTypes.STRING,
    title: DataTypes.STRING,
    authors: DataTypes.STRING,
    year: DataTypes.INTEGER,
    quantity: DataTypes.INTEGER,
    organizationId: DataTypes.INTEGER,
    timesReserved: DataTypes.INTEGER,
    deleted: DataTypes.BOOLEAN
  }, {
    sequelize,
    modelName: 'Book',
  });
  return Book;
};
